spec/support/features.rb

```diff
@@ -7,11 +7,23 @@ def in_spec?
 end
 
 def with_env_vars_runner(env)
+  # hold on to old values
   env = env.stringify_keys
```

Comment:
these helper methods are neat !! 

Have you considered using `ClimateControl` instead? https://github.com/thoughtbot/climate_control - I think it basically does what we are doing here but I am not sure it provides the ability to delete an `ENV` var so maybe we should stick with what we have